lib/puppet/provider/cisco_interface_ospf/cisco.rb

```diff
@@ -31,29 +31,28 @@
 
   mk_resource_methods
 
-  # Getter properties that need to generate the getter functions
-  GETTER_PROPS = [
-    :hello_interval, :dead_interval
+  INTF_OSPF_NON_BOOL_PROPS = [
+    :cost,
+    :hello_interval,
+    :dead_interval,
+    :message_digest_key_id,
+    :message_digest_algorithm_type,
+    :message_digest_encryption_type,
+    :message_digest_password,
+    :area,
```

Comment:
nit: we like to keep the properties in alpha order, except when they need to process in a certain order -- in which case we call out why it's misordered. Does `area` need special handling?